lesson_15/vicentevigueras/ts-node-express/src/index.ts

```diff
@@ -0,0 +1,14 @@
+// src/index.js
+import express, { Express, Request, Response } from "express";
+import dotenv from "dotenv";
+import path from "path";
+
+dotenv.config();
+
+const app: Express = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.static('/workspaces/code-differently-24-q1/lesson_01/VicenteVigueras'));
```

Comment:
Should copy the files into a sub-folder in "ts-node-express". Convention usually dictates this folder be named "public", but fine if you use a different name.

https://dev.to/mr_ali3n/folder-structure-for-nodejs-expressjs-project-435l

Reply:
I think it's fixed, let me know if I have to change anything.